examples/highfreq/high_freq_tree.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
Why is this file necessary?

Reply:
This file is a demo for high-frequency workflow to run in Jupiter notebook and I think it might be straightforward for others to have a python demo.

Reply:
Since it is really similar to the current workflow example and I will upload a YAML version of it later,   we can also delete it to make the example more concise, depending on whether it is necessary to have a high-frequency version of workflow.